### PR TITLE
Detomatix no-self-explode

### PR DESCRIPTION
**Pull request by SonixApache**
image
dreamseeker_2020-12-08_21-06-17

TODO:
 - [ ] some of the garbled messages actually break the (Reply) button... should I fix that?
 - [x] update wiki post-merge

🆑 
 - tweak: Detomatix: Rather than blowing YOU up, failing a chance roll now outs you with a PDA message.
 - tweak: Detomatix: Trying to blow up a fellow syndie still fails, but sends them a code word and shows you a code response instead of a blank error.

### CODE REVIEW

**Comment by Manleto** [comment]
AI's shouldn't even show up because they don't have a physical PDA at all

**Comment by SonixApache** [comment]
Yeah, I'm very on the fence about the AI / captain deto removal option. Maybe I should put it back.

**Comment by Kurfursten** [comment]
Just don't list them if they are guaranteed to fail. Newbie traps are bad.

**Comment by SonixApache** [comment]
deleted

**Comment by APCUnlocked** [comment]
"tweak: Detomatix: Trying to blow up a fellow syndie still fails, but sends them a code word and shows you a code response instead of a blank error."

Time to start paying close attention to the PDA messaging console if it logs those messages

**Comment by TGFH** [comment]
Why?

**Comment by gurfan** [comment]
because spending 8 TC on an item that has a high chance of destroying your pda and hurting you the moment you try to use it is fucking gay

**Comment by gurfan** [comment]
because spending 8 TC on an item that has a high chance of destroying your pda and hurting you the moment you try to use it is fucking gay

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 1]
```diff
@@ -2370,6 +2385,7 @@ var/global/list/obj/item/device/pda/PDAs = list()
 		if(P.show_overlays)
 			P.overlays += image('icons/obj/pda.dmi', "pda-r")
 	else
+		to_chat(world,"unstable")
```
Comment (gurfan) [inline code/game/objects/items/devices/PDA/PDA.dm]: huh

Comment (SonixApache) [reply code/game/objects/items/devices/PDA/PDA.dm]: hmmm

**Comment by StreptoMutans** [comment]
Ah yes, time to antagcheck once you capture an unlocked PDA

**Comment by Kurfursten** [comment]
That's a very valid concern, although on the other hand randomly blowing up PDAs as a non antag is a sure way to get banned. But why even create the incentive? I think the idea probably has to be axed.

**Comment by SonixApache** [comment]
That functionality is already present. Trying to blow up a PDA with an uplink results in an error.
The only change this makes is that you now send a message to the person you antagcheck.

I think leaving it in and banning people that blow up random PDAs in an attempt to find a syndie is a better idea.

**Comment by TGFH** [comment]
> 
> 
> because spending 8 TC on an item that has a high chance of destroying your pda and hurting you the moment you try to use it is fucking gay

Never had a problem with it as a tator myself. PDA bombs are pretty powerful and the tradeoff is an acceptable risk for something that can potentially kill someone, deafens them, and will probably delete their ID.

**Pull request merged**